package.json

```diff
@@ -5,15 +5,15 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "watcher:signature-request": "node src/watcher.js signature-request-watcher.config.js",
-    "watcher:collected-signatures": "node src/watcher.js collected-signatures-watcher.config.js",
-    "watcher:affirmation-request": "node src/watcher.js affirmation-request-watcher.config.js",
-    "sender:home": "node src/sender.js home-sender.config.js",
-    "sender:foreign": "node src/sender.js foreign-sender.config.js",
+    "watcher:signature-request": "node src/watcher.js signature-request-watcher.config.js | tee -a logs/watcher-signature-request.txt | pino",
```

Comment:
Is it possible to enable this only for a `dev` (or `testing`) environment? It is assuming that the bridge will be run by `systemd` facility on a production system this means that logs will be always redirected to `rsyslogd` and stored in the log files automatically.

Reply:
Oh, yes. I will think a little about the best way to handle this. Some options are piping to an ad hoc module that prettifies (or doesn't) based on the `NODE_ENV` variable, or just removing the prettifier and making this a responsibility of the caller.